code/datums/components/crafting/recipes.dm

```diff
@@ -158,16 +158,15 @@
 
 /datum/crafting_recipe/ed209
 	name = "ED209"
-	result = /mob/living/simple_animal/bot/ed209
+	result = /mob/living/simple_animal/bot/secbot/ed209
 	reqs = list(/obj/item/robot_suit = 1,
 				/obj/item/clothing/head/helmet = 1,
 				/obj/item/clothing/suit/armor/vest = 1,
 				/obj/item/bodypart/l_leg/robot = 1,
 				/obj/item/bodypart/r_leg/robot = 1,
 				/obj/item/stack/sheet/iron = 1,
 				/obj/item/stack/cable_coil = 1,
-				/obj/item/gun/energy/e_gun/dragnet = 1,
-				/obj/item/stock_parts/cell = 1,
+				/obj/item/gun/energy/disabler = 1,
```

Comment:
Dragnet requirement changed to disabler?

Reply:
ED-209s use disablers. They don't use DRAGnets.

Reply:
You didn't mention you made this change anywhere in the PR body or changelog

Reply:
> You didn't mention you made this change anywhere in the PR body or changelog



> * misc fixes

![it didnt even check for it](https://user-images.githubusercontent.com/62388554/218544270-51d775a6-2801-471b-8af8-c71c2a0b402a.png)

Please read the code. 

It uses a disabler in the recipe, so I dont understand what you are fussing about